### DISCUSSION TITLE
How modify color components?

### DISCUSSION THREAD

**Discussion by aldoprogra** [directus-labs/agency-os#115] [General]
Hi I would like to modify the colors of buttons, the background of the portal, but I struggle to understand how. Even from by modifying the globals from directus seems not working, and when I dive in the components like UCard seems compiled and no modifiable. Someone knows how to personalize most of fall the grey portal background?

**Comment by Abdallah-Awwad** [comment]
Sorry for the late reply, I think you can change it from [theme.ts](https://github.com/directus-labs/agency-os/blob/78d36dddf74289719249053165cd879cf9fdbeea/theme.ts#L17C1-L18C16) you can replace the it by other color names. 
If you want to use specific hex color code, you need to refer to tailwind to [customize colors](https://tailwindcss.com/docs/customizing-colors)
